test/functional/outputs/otlp/app-log-sample.json

```diff
@@ -0,0 +1,149 @@
+{
+  "resourceLogs": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "node.name",
+            "value": {
+              "stringValue": "ip-10-0-47-200.ec2.internal"
+            }
+          },
+          {
+            "key": "cluster.id",
+            "value": {
+              "stringValue": "functional"
+            }
+          },
+          {
+            "key": "k8s.pod.name",
+            "value": {
+              "stringValue": "functional"
+            }
+          },
+          {
+            "key": "k8s.pod.uid",
+            "value": {
+              "stringValue": "e44c037f-f335-47a5-aa9a-6b81a8fb5193"
+            }
+          },
+          {
+            "key": "k8s.container.name",
+            "value": {
+              "stringValue": "collector"
+            }
+          },
+          {
+            "key": "k8s.container.id",
+            "value": {
+              "stringValue": "cri-o://67b1c8b2da26370704b421cd9c3b90e15154c402b330367f353cb2652411332b"
+            }
+          },
+          {
+            "key": "k8s.namespace.name",
+            "value": {
+              "stringValue": "test-ofuix3pj"
+            }
+          },
+          {
+            "key": "k8s.pod.label.test-client",
+            "value": {
+              "stringValue": "true"
+            }
+          },
+          {
+            "key": "k8s.pod.label.testname",
+            "value": {
+              "stringValue": "functional"
+            }
+          },
+          {
+            "key": "k8s.pod.label.testtype",
+            "value": {
+              "stringValue": "functional"
+            }
+          }
+        ]
+      },
+      "scopeLogs": [
+        {
+          "scope": {},
+          "logRecords": [
+            {
+              "timeUnixNano": "1693227568573159188",
+              "observedTimeUnixNano": "1719950665726889376",
+              "severityNumber": 9,
+              "body": {
+                "stringValue": "Format me to a container message!"
+              },
+              "attributes": [
```

Comment:
These should be resource scoped

Reply:
Please explain or elaborate a bit.

Reply:
You are correct that source should be resource scoped.   Will fix that.   Here is the latest example:  
https://github.com/openshift/cluster-logging-operator/blob/master/hack/logsamples/otlp_container.json